src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -893,7 +893,7 @@ def _prepare_deployment_properties_unmodified(cmd, deployment_scope, template_fi
         template_obj = _remove_comments_from_json(_urlretrieve(template_uri).decode('utf-8'), file_path=template_uri)
     elif template_spec:
         template_link = TemplateLink(id=template_spec, mode="Incremental")
-        template_obj = show_resource(cmd=cmd, resource_ids=[template_spec]).properties['template']
+        template_obj = show_resource(cmd=cmd, resource_ids=[template_spec]).properties['main_template']
```

Comment:
May I ask why we need to modify the property name in `properties`?

Reply:
It is a breaking change in the SDK (https://github.com/Azure/sdk-release-request/issues/1469). Template has now been changed to main_template.

Reply:
Got it, thanks